tests/test_install.py

```diff
@@ -7,7 +7,7 @@
 
 
 @skip_windows(reason='Pythonz is unsupported')
-@pytest.mark.skipif(sys.version_info > (2,7) and os.environ.get('CI') == 'true',
+@pytest.mark.skipif(sys.platform == 'cygwin' or (sys.version_info > (2,7) and os.environ.get('CI') == 'true'),
```

Comment:
I think It's better to separate `sys.platform == 'cygwin'` in another `skipif`, so the reason for skipping that would be explicit